tailormap-components/projects/core/src/styles/styles.css

```diff
@@ -68,7 +70,7 @@
   right: 10px;
   top: 10px;
   bottom: 10px;
-  width: 300px;
+  width: calc(var(--overlay-panel-columns) * var(--overlay-panel-width));
```

Comment:
Wouldn't do this in the styles.css because they are meant to be global and this is quite specific